frontend/lib/src/assets/css/vega-tooltip.scss

```diff
@@ -29,7 +29,7 @@
   visibility: hidden;
   padding: 8px;
   position: fixed;
-  z-index: 1000;
+  z-index: 1000050;
```

Comment:
Might be worthwhile to put a comment in the zIndices file to check other places for z-index to determine the right value.